### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_ScreenReader: NVDA is announcing the information "Discover the best customizations to make VS Code yours" twice.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#7831]
--- GitHub Tags:
-A11yMAS; -A11yTCS; -A11ySev3; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2022; -WCAG1.3.1; -Info and Relationships; -Element:Button; -NVDA

--- Environment and OS details:
Visual Studio Code
Windows 11 Pro version 21H2 (OS Build 22000.856)

--- Repro Steps:
1)Open Visual studio code editor
2)Using "Alt+H" key navigate to the "Help" menu bar option.
3)Using "Down" arrow key navigate to the "Welcome" item.
4) Make sure you have the following setting in settings.json: "workbench.startupEditor": "gettingStarted"
5)Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started" and observe that the NVDA is announcing the information "Discover the best customizations to make VS Code yours" twice or not.

--- Actual Result:
NVDA is announcing the information "Discover the best customizations to make VS Code yours" twice.

--- Expected Result:
NVDA should not announce the "Discover the best customizations to make VS Code yours" twice it should be announced only once.
